static/audiofeed.yaml

```
@@ -1,3 +1,28 @@
+- title: 'Der M10Z Comic Cast 02 - Ein Videospielsachcomic'
+  date: 2025-06-23T15:00
+  image: https://raw.githubusercontent.com/LucaNerlich/m10z/main/static/img/formate/cover/comic-cast.jpg
+  description: |2-
+                Moin meine Freund:innen,
+
+                zum zweiten mal könnt ihr unseren Comic Cast hören. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)